WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+Localization.swift

```diff
@@ -26,10 +26,15 @@ extension GutenbergViewController {
     }
 
     private func currentLProjFolderName(in bundle: Bundle) -> String? {
-        /// Localizable.strings file path use dashes for languages and regions (e.g. pt-BR)
-        /// We cannot use Locale.current.identifier directly because it uses underscores
-        /// Bundle.preferredLocalizations matches what NSLocalizedString uses
-        /// and is safer than parsing and converting identifiers ourselves
+        // Localizable.strings file path use dashes for languages and regions (e.g. pt-BR)
+        // We cannot use Locale.current.identifier directly because it uses underscores
+        // Bundle.preferredLocalizations matches what NS-LocalizedString uses
+        // and is safer than parsing and converting identifiers ourselves.
+        //
+        // Notice the - in the NSLocalized... method. There seem to be a bug in genstrings where
+        // it tries to parse lines coming from comments, too:
```

Comment:
Yep… known `genstrings` bug… 😞

Reply:
No I don't think it is. And that's also the reason why we need to exclude `AppLocalizedStrings.swift` file from `genstrings` / `ios_generate_strings_file_from_code` parsing so that `genstrings` doesn't pick up the `func AppLocalizedString` function definition thinking it's a function call.

I'd be great if they reimplemented `genstrings` on top of the [new Swift Syntax parser](https://forums.swift.org/t/a-new-swift-parser-for-swiftsyntax/59813) — well though I guess it'd still have to support ObjC too and need to also use a dedicated ObjC parser lib for this… so probably not as trivial) — instead of relying to Regular Expressions (at least I think that's what `genstrings` use under the hood, given that bug)… but alas…